### PR TITLE
Add comment property for create table

### PR DESCRIPTION
**Pull request by yuananf**
Replaced #3193

### CODE REVIEW

File: presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java [hunk 1]
```diff
@@ -641,7 +641,8 @@ protected Void visitCreateTable(CreateTable node, Integer indent)
                     .append(" (");
 
             Joiner.on(", ").appendTo(builder, transform(node.getElements(),
-                    element -> element.getName() + " " + element.getType()));
+                    element -> element.getName() + " " + element.getType()
+                             + (element.getComment().isPresent() ? " comment '" + element.getComment().get() + "'" : "")));
```
Comment (electrum) [inline presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java]: Make "comment" uppercase

Comment (electrum) [inline presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java]: We also need to escape the value:
```java
element -> {
    String value = element.getName() + " " + element.getType();
    if (element.getComment().isPresent()) {
        value += " COMMENT " + formatStringLiteral(element.getComment().get());
    }
    return value;
}

**Comment by electrum** [comment]
This seems reasonable overall, though I don't like putting the comment in `HiveColumnHandle` since it doesn't need to be serialized and passed around for every read. Though maybe that's not a problem. We will need to think about it.

Can you split out the Hive changes into a second commit?

**Comment by yuananf** [comment]
@electrum , comments addressed.

**Comment by cpcloud** [comment]
This would be swell to have. @yuananf What's the status here?

**Comment by cpcloud** [comment]
@electrum What's missing from this PR? Happy to take it over.

**Comment by yuananf** [comment]
@cpcloud this PR waiting for review, I'll rebase it on current master.

**Comment by yuananf** [comment]
rebased

File: presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java [hunk 2]
```diff
@@ -115,6 +119,11 @@ public ColumnMetadata getColumnMetadata(TypeManager typeManager)
         return new ColumnMetadata(name, typeManager.getType(typeName), null, isHidden());
     }
```
Comment (damiencarol) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java]: Missing @JsonProperty ?

Comment (yuananf) [inline presto-hive/src/main/java/com/facebook/presto/hive/HiveColumnHandle.java]: Nice catch! Fixed.

**Comment by electrum** [comment]
Sorry for the long delay on this. Will merge this shortly.

**Comment by electrum** [comment]
Merged, thanks!

**Comment by yuananf** [comment]
Thanks!

**Comment by privateproperty123** [comment]
More info about sell property please visit link : - [http://privatepropertybuyers.co.uk/](http://privatepropertybuyers.co.uk/) .

**Pull request closed without merging**